kolejka/foreman/foreman.py

```diff
@@ -23,14 +23,22 @@
 from kolejka.common import KolejkaTask, KolejkaResult, KolejkaLimits
 from kolejka.common import MemoryAction, TimeAction, parse_memory
 from kolejka.client import KolejkaClient
+from kolejka.common.images import (
+    pull_docker_image,
+    get_docker_image_size,
+    check_docker_image_existance,
+    list_docker_images,
+    remove_docker_image,
+    docker_build_local_image
+)
 from kolejka.worker.stage0 import stage0
 from kolejka.worker.volume import check_python_volume
 
-def manage_images(pull, size, necessary_images, priority_images):
+def manage_images(pull, size, necessary_images, priority_images, base_images):
```

Comment:
Think about limiting storage for Foreman when additional images are created. @grzegorz-gutowski